packages/styles/button.css

```diff
@@ -41,6 +41,10 @@
   gap: 8px;
 }
 
+.Button--tag {
+  position: relative;
+}
```

Comment:
Why `position: relative`?

Reply:
That is needed for hover to work. Since the `:before` element has `position: absolute`.